pkg/dispatcher/operator/calculations/controller.go

```diff
@@ -30,9 +30,8 @@ const (
 )
 
 var calculationValues = prometheus.NewGaugeVec(prometheus.GaugeOpts{
-	Namespace: "vega",
-	Name:      "vega_calculations",
-	Help:      "Calculation ID, status and time of creation",
+	Name: "vega_calculations",
```

Comment:
let's keep the namespace and change only the name from `vega_calculations` to `calculations`